drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -5837,8 +5837,8 @@ bool RasterizerStorageGLES2::free(RID p_rid) {
 		memdelete(t);
 		render_target_owner.free(p_rid);
 		memdelete(rt);
-
 		return true;
+
 	} else if (texture_owner.owns(p_rid)) {
```

Comment:
These minor formatting changes are considered 'noise' in the commit and these should be removed, in general the policy is to only change the places you are changing code.